librrd.h

```diff
@@ -126,3 +130,8 @@ int             rrd_del_src(RRD_PLUGIN * plugin, RRD_SOURCE * source);
  * be called every 5 seconds by the client,
  */
 int             rrd_sample(RRD_PLUGIN * plugin);
+
+
+JSON_Value *json_for_source(RRD_SOURCE * source);
+
+JSON_Value *json_for_plugin(RRD_PLUGIN * plugin);
```

Comment:
I know, But Tests needs to use them else i have to write them again in the tests. Is there any workaround.